lib/jobs/ServiceInstanceUpdateJob.js

```diff
@@ -118,12 +118,25 @@ class ServiceInstanceUpdateJob extends BaseJob {
       });
   }
 
-  static getOutdatedDiff(deploymentName, plan) {
+  static createOpts(instanceId) {
```

Comment:
If required you can remove the wrapper method `createOpts` and move it to cloudController with the name `getOrgAndSpaceGuid`, so that way you can reuse the same from controller as well as jobs. We move reusable to higher levels of common points based on the places from where it can be consumed. So looking at this methods current usage, to avoid dupe, its best we move this method to cloudController directly.